client/src/App.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import {
+  BrowserRouter as Router,
+  Route,
+  Switch,
+  Redirect,
+} from 'react-router-dom';
+
+import * as Routes from './routes';
+
+import Home from './pages/Home';
+import Member from './pages/Member';
+import Login from './pages/Login';
+
+import Navbar from './components/navbar/Navbar';
+
+import {
+  endUserSession,
+  getUserAuth,
+  startUserSession,
+} from './utils/apiWrapper';
+
+function App() {
+  const [isAuthenticated, setIsAuthenticated] = React.useState(false);
```

Comment:
nit: you can do `import React, {useEffect, useState} from 'react'` as the top so you don't need to include `React.` in front of `useState` and `useEffect`

Reply:
This was more of a style thing, I kind of liked :), I'll change it to other way.

Reply:
Ah I see, honestly it doesn't matter that much as long as it's consistent :) i think I've seen it imported at the top more frequently so wanted to stick with that